static/js/billing/helpers.js

```diff
@@ -7,12 +7,12 @@ exports.create_ajax_request = function (
     numeric_inputs = [],
     redirect_to = "/billing",
 ) {
-    const form = $("#" + form_name + "-form");
-    const form_loading_indicator = "#" + form_name + "_loading_indicator";
-    const form_input_section = "#" + form_name + "-input-section";
-    const form_success = "#" + form_name + "-success";
-    const form_error = "#" + form_name + "-error";
-    const form_loading = "#" + form_name + "-loading";
+    const form = $(`#${CSS.escape(form_name)}-form`);
+    const form_loading_indicator = `#${CSS.escape(form_name)}_loading_indicator`;
+    const form_input_section = `#${CSS.escape(form_name)}-input-section`;
+    const form_success = `#${CSS.escape(form_name)}-success`;
+    const form_error = `#${CSS.escape(form_name)}-error`;
+    const form_loading = `#${CSS.escape(form_name)}-loading`;
```

Comment:
@hackerkid can you do manual testing on the billing conversion here?

Reply:
I couldn't find any regression in manual testing of /billing and /upgrade pages.